lib/langchain.rb

```diff
@@ -1,8 +1,22 @@
 # frozen_string_literal: true
 
+require "logger"
+
 require_relative "./version"
 require_relative "./dependency_helper"
-require_relative "./logging"
+module Langchain
+  class << self
+    attr_accessor :default_loaders
+    attr_accessor :logger
+
+    attr_reader :root
+  end
+
+  @default_loaders ||= []
```

Comment:
What do you think about loading all of the default Loaders right out of the gate?

Reply:
I was thinking we should indeed include a standard set of default loaders. Maybe not ALL of them, but a reasonable set of common ones.

Reply:
Yeah, at least the ones for the most common file extensions.